### ISSUE TITLE
Examples Fail

### ISSUE THREAD

**Issue by backjoob** [balde/balde#22]
I followed instructions here: https://balde.rgm.io/doc/getting-started.html

I built balde with examples, but running them as specified in the above documentation gives the following:
___________________________________________________________
balde-0.1.2/examples$ ./hello --runserver
Status: 405 Method Not Allowed
Content-Type: text/plain; charset=utf-8
Content-Length: 73

405 Method Not Allowed

The method is not allowed for the requested URL.
___________________________________________________________

with the exception of hello-with-static which gives the following:
___________________________________________________________
balde-0.1.2/examples$ ./hello-with-static --runserver
Status: 404 Not Found
Content-Type: text/plain; charset=utf-8
Content-Length: 136

404 Not Found

The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.
___________________________________________________________

The same errors occur with or without --runserver.

**Comment by rafaelmartins** [comment]
hi. the documentation is for master branch, as stated in the website header. if you install it from the master branch all the examples should work. I'm planning a new release, but had no time to work on it yet.

**Comment by backjoob** [comment]
Thanks. I got examples working from master. It seems that peg/leg are necessary to compile, correct?

The only example that throws an error is hello-with-static, but I don't know enough to tell whether that's what it's meant to do.

If you have time to put a quickstart in the README that would be great. Like prereqs git clone...; ./autogen.sh; ./configure; make; make install

**Comment by rafaelmartins** [comment]
yes, peg/leg is needed for git. distfiles with prebuilt grammar can be found here: https://distfiles.rgm.io/balde/LATEST/, forgot to mention.

about that example, if you hit `/static/foo.js`, `/static/foo.css` and `/static/asd/bola.txt` with your browser you'll see the static files, that are embedded and served directly from the binary

the docs should get update soon. there are quite a few undocumented options too, that need work.

Thanks
